src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -18,6 +18,7 @@
     public final static String JSON_DIRECTORY = "json/";
 
     protected static final String SAMPLE_JSON = "sample.json";
+    protected static final String SAMPLE_XML_JSON = "sample-xml.json";
```

Comment:
you don't need to add json to test it, you can test the code as unit test rather than integration